src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -77,11 +77,11 @@
         DESC_BOB = new EditPersonDescriptorBuilder().withName(VALID_NAME_BOB)
                 .withPhone(VALID_PHONE_BOB).withEmail(VALID_EMAIL_BOB).withAddress(VALID_ADDRESS_BOB)
                 .withTags(VALID_TAG_HUSBAND, VALID_TAG_FRIEND).build();
-        DESC_AMY_ORDER = new EditOrderDescriptorBuilder().withName(VALID_NAME_AMY)
-                .withPhone(VALID_PHONE_AMY).withAddress(VALID_ADDRESS_AMY)
+        DESC_AMY_ORDER = new EditOrderDescriptorBuilder()
```

Comment:
Will it be possible to use stubs or any other methods to illustrate the relationship between Order and Person for testing purposes?

Reply:
Will have a look at this when im updating the tests,
For now ive just updated the tests to ensure that they all pass